.github/workflows/update-viewer.yml

```diff
@@ -0,0 +1,28 @@
+name: Update viewer.php - viewer.html got updated so viewer.php needs an update
```

Comment:
Maybe the name should be only `Update viewer.php` and the extended description should be left for the failed step?

Or rathar than `Update viewer.php` maybe `Check viewer.php`, as the workflow only checks it, but the update itself needs to be done by a human (at least, for now :-P ). If this is applied the file should be renamed to `check-viewer.yml`.

Reply:
done